test/graphql-schema-gen/schema.graphql

```diff
@@ -1163,13 +1156,13 @@ input PayloadPreference_where {
   updatedAt: PayloadPreference_updatedAt_operator
   createdAt: PayloadPreference_createdAt_operator
   id: PayloadPreference_id_operator
-  OR: [PayloadPreference_where_or]
   AND: [PayloadPreference_where_and]
+  OR: [PayloadPreference_where_or]
 }
 
 input PayloadPreference_user_Relation {
   relationTo: PayloadPreference_user_Relation_RelationTo
-  value: String
+  value: JSON
```

Comment:
Is this a change from your updates or was this not generated before?

Reply:
@Dan This was me regenerating the types with the updated schema changes. Since a few of test suites had graphql schemas. Technically, they would be outdated after my changes.

Reply:
Confirmed, this change happens on `main` as well. So, unrelated to this PR.